App/Containers/PhotoScreen.tsx

```diff
@@ -41,7 +41,7 @@ class PhotoScreen extends React.Component<StateProps & NavigationScreenProps<{}>
           item={{type: 'photo', photo: this.props.photo, threadId: this.props.threadId, threadName: this.props.threadName}}
           onComment={this.onComment}
           onLikes={this.onLikes}
-          recentCommentsCount={5}
+          recentCommentsCount={-1}
```

Comment:
Just using an optional for `ThreadDetailCard.props.recentCommentsCount` and then not setting here would be nice.

Reply:
updated